app/controllers/comments_controller.rb

```diff
@@ -42,7 +42,7 @@ def comment_params
   end
 
   def set_resource
-    resource = comment_params[:resource_name].singularize.camelize.constantize.find_by(id: comment_params[:resource_id])
+    resource = comment_params[:resource_name].singularize.camelize.constantize.find_by(id: comment_params[:resource_id]) rescue nil
```

Comment:
rescue is for situations where the result of `[:resource_name].singularize.camelize` is not a constant. Or if it is, not an ActiveRecord model. Could be caused by an invalid `resource_name` passed in